contracts/adapters/AaveV3LeverageStrategyExtension.sol

```diff
@@ -31,14 +37,22 @@ import {IBaseManager} from "../interfaces/IBaseManager.sol";
  *
  */
 contract AaveV3LeverageStrategyExtension is AaveLeverageStrategyExtension {
+
+    uint8 public currentEModeCategoryId;
+    IPoolAddressesProvider public lendingPoolAddressesProvider;
+    uint256 public maxOraclePriceAge;
+    bool public overrideNoRebalanceInProgress;
+
     constructor(
         IBaseManager _manager,
         ContractSettings memory _strategy,
         MethodologySettings memory _methodology,
         ExecutionSettings memory _execution,
         IncentiveSettings memory _incentive,
         string[] memory _exchangeNames,
-        ExchangeSettings[] memory _exchangeSettings
+        ExchangeSettings[] memory _exchangeSettings,
+        IPoolAddressesProvider _lendingPoolAddressesProvider,
+        uint256 _maxOraclePriceAge
```

Comment:
In the deployment we will have to set this parameter to a reasonable value. Meaning the maximum "age" / "delay" of price updates to be expected on chainlink.
@FlattestWhite @pblivin0x